src/main/java/org/com/itpple/spot/server/domain/user/api/UserInfoController.java

```diff
@@ -52,4 +52,11 @@ public ResponseEntity<UserInfoResponse> updateUserInfo(
         Long userId = customUserDetails.getUserId();
         return ResponseEntity.ok(userInfoService.updateUserInfo(userId, userInfoRequest));
     }
+
+    //다른 유저 프로필 조회
+    @GetMapping("/{userid}")
+    public ResponseEntity<UserInfoDto> getUserProfile(@PathVariable Long userid) {
```

Comment:
```suggestion
    public ResponseEntity<UserInfoDto> getUserProfile(@PathVariable Long userId) {
```
camelCase로 적으면 좋을 거 같아요